### PR TITLE
Added xpresso dataloader

### PR DESCRIPTION
**Pull request by haimasree**
Of course I will remove the kipoiseq branch from model.yaml once https://github.com/kipoi/kipoiseq/pull/96 is merged and a new package version is released.

### CODE REVIEW

File: Xpresso/model-template.yaml [hunk 1]
```diff
@@ -7,13 +7,8 @@ args:
...
 default_dataloader:
-    defined_as: kipoiseq.dataloaders.SeqIntervalDl
-    default_args:
-        auto_resize_len: 10500
-        alphabet_axis: 1
-        alphabet: "ACGT" # Order of letters in 1-hot encoding
-        use_strand: False
-        ignore_targets: False # if True, dont return any target variables
+    defined_as: kipoiseq.dataloaders.AnchoredGTFDl
+
```
Comment (Avsecz) [inline Xpresso/model-template.yaml:11]: Please add default_args for num_upstream, num_downstream. Probably need to fix `override_default_kwargs` https://github.com/kipoi/kipoi/blob/c1694eb5a0f7ad4607edcf389dcc927a75cf15b4/kipoi/specs.py#L816 to pin args that don't have a default value yet.

**Review by Avsecz — CHANGES_REQUESTED** [review]

File: Xpresso/model-template.yaml [hunk 2]
```diff
@@ -32,9 +27,11 @@ dependencies:
       - numpy=1.19.2
       - pip=20.3.3
       - pysam=0.15.3
+      - cython=0.29.23
     pip:   # install via pip
       - keras==2.4.3
       - tensorflow==2.4.1
+      - git+https://github.com/kipoi/kipoiseq.git@add-xpresso-dataloader
```
Comment (Avsecz) [inline Xpresso/model-template.yaml:34]: After kipoiseq will get merged and released to pypi, then specify `kipoiseq==pypi_version`

Comment (haimasree) [reply Xpresso/model-template.yaml:34]: I have merged the pr but keeping this branch alive until the package is released. I will speak to Flo about this.

File: Xpresso/model-template.yaml (hunk 1 above)
Comment (haimasree) [reply Xpresso/model-template.yaml:11]: I have pinned the values in the yaml docstring [here](https://github.com/kipoi/kipoiseq/blob/master/kipoiseq/dataloaders/sequence.py#L407-L412).
The bug will be fixed here - https://github.com/kipoi/kipoi-utils/issues/28

**Pull request merged**